stanford-earth/matson
# EARTH-86

# READY FOR REVIEW

# Summary
- Various homepage specific and paragraph tweaks
- Add body classes for front page and not front pages

# Needed By (Date)
- N/A

# Urgency
- Not critical

# Steps to Test

1. Pull this branch
2. Ensure that classes are dynamically being added to the <body>
3. See various style tweaks

# Affected Projects or Products
- Earth
- Matson

# Associated Issues and/or People
- https://stanfordits.atlassian.net/browse/EARTH-86

# See Also
- [PR Checklist](https://gist.github.com/sherakama/0ba17601381e3adbe0cad566ad4d80a5)
